packages/mobile/src/components/block-messages/BlockMessages.tsx

```diff
@@ -131,23 +142,25 @@ export const BlockMessagesDrawer = () => {
           <Text style={styles.title}>{messages.title}</Text>
         </View>
         <Text style={styles.confirm}>
-          {messages.confirmText1}
-          {user?.name}
-          {messages.confirmText2}
+          {doesBlockUser
```

Comment:
nit: not sure if what im about to suggest is cleaner, so fine with what you decide, but seeing a lot of `doesBlockUser` relate conditions below. maybe it makes sense to have var e.g. message,title,variant in a colocated space here and then use below

Reply:
Hm good call. I think if we had more than 2 possible states it would def make sense, but since it's only ever block/unblock here I think this is fine.